tools/autograd/derivatives.yaml

```diff
@@ -951,6 +951,10 @@
 - name: binary_cross_entropy(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean) -> Tensor
   self: binary_cross_entropy_backward(grad, self, target, weight, reduction)
 
+- name: binary_cross_entropy_backward(Tensor grad_output, Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean) -> Tensor
+  self: binary_cross_entropy_double_backward(grad_output, grad, self, target, weight, reduction)
+  grad_output: binary_cross_entropy_double_backward_grad_output(grad_output, self, target, weight, reduction)
```

Comment:
this doesn't look correct -- for a pointwise function, you probably don't need the original grad_output (because you are taking the derivative wrt to it, so that term is just going to fall out.

I also think you could probably reuse the computation for the single-backward here if you want, with just passing in a tensor of 1s instead of the actual grad_output.